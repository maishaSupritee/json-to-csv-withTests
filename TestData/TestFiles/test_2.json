{
  "node": [
    {
      "A": "te\nst",
      "C": "te\tst"
    },
    {
      "C": "x",
      "C": null
    }
  ]
}
