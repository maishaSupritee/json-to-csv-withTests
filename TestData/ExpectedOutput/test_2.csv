"node_A","node_C"

"te
st","te	st"

"","None"

